### PR TITLE
fix: moved to API 3 github

### LINKED ISSUE #116: Move to API v3 version to update membership
Currently, we use org_id and team_id to update membership which was the only way when this project was created, the new API v3 enables using team slug and org username for the same/
Ref: https://developer.github.com/v3/teams/members/#add-or-update-team-membership-for-a-user

```
PUT /orgs/:org/teams/:team_slug/memberships/:username
```

### PR DESCRIPTION
**Pull request by AmanRaj1608**
Fixes #116 

### Description

Removing dependency of org_id and team_id using new v3 API.

### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [x] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [x] Documentation

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -157,8 +158,8 @@ const addMember = data => {
   const { email, username } = data
   const regex = /^20\d{7}@iiitv(adodara)?.ac.in$/; // eslint-disable-line
   const promise = new Promise((resolve, reject) => {
-    const pref = regex.test(email) ? 'batch-of-' + parseInt(parseInt(email.substring(0, 4)) + 4) : 'outsiders'
-    const url = `https://api.github.com/orgs/iiitv/teams/${pref}/memberships/${username}`
+    const pref = regex.test(email) ? `batch-of-${parseInt(email.substring(0, 4)) + 4}` : 'outsiders'
```
[the remark this answers is not in the document]
Comment (AmanRaj1608) [reply server.js]: Sorry made PR.
Reverting back.

**Comment by AmanRaj1608** [comment]
I updated Glitch manually
Then tested by adding myself with my personal email.

I was added in outsiders too

image

@thepushkarp  @yashshah2820 Confirm if this is valid for other id's too

**Comment by aashutoshrathi** [comment]
Also try leaving and adding yourself back to batch-of-2022

**Review by yashshah2820 — APPROVED** [review]

**Comment by yashshah2820** [comment]
> @thepushkarp @yashshah2820 Testing and approvals needed!

Done sir, I removed myself from outsiders team and I was added once again using personal email id 👍

**Comment by aashutoshrathi** [comment]
@yashshah2820 I repeat not the outsiders try on  batch-of-x

**Comment by thepushkarp** [comment]
Yes, It's working for batch-of-x. I got added from GetMeIn after removing myself.

**Comment by thepushkarp** [comment]
Here's the mail received

**Review by aashutoshrathi — APPROVED** [review]
GGWP

**Review by thepushkarp — APPROVED** [review]

**Comment by aashutoshrathi** [comment]
@yashshah2820 Update glitch deployment using import feature.

**Comment by yashshah2820** [comment]
done sir

**Pull request merged**